frontend/src/components/samples/details/SampleDetailsContent.js

```diff
@@ -178,6 +178,15 @@ const SampleDetailsContent = ({samplesByID, sampleKindsByID, containersByID, pro
               </Descriptions.Item>
               <Descriptions.Item label="Coordinates">{sample.coordinates || "—"}</Descriptions.Item>
               <Descriptions.Item label="Comment" span={3}>{sample.comment}</Descriptions.Item>
+              { // For flags there's a difference between having a false value (QC failed)
+                // and a null value (QC not performed yet)
+              }
+              <Descriptions.Item label="Quantity Flag">
```

Comment:
Also maybe create a component akin <SampleDepletion depleted=BOOL /> but that offer more options to be able to reuse it for the library QC and the list. I could see something like <QcFlag passed=[{key: BOOL}, {key: BOOL},...], tooltip=BOOL /> this would combine a number of flags into a single icon with or without tooltip giving the details by key.